test/e2e/helpers/visu/bpmn-page-utils.ts

```diff
@@ -150,50 +147,46 @@ export class BpmnPageSvgTester extends PageTester {
     });
   }
 
-  async expectLabel(bpmnId: string, expectedText?: string): Promise<void> {
-    if (!expectedText) {
-      return;
-    }
-    const labelLastDivElementHandle = await this.currentPage.waitForSelector(this.bpmnQuerySelectors.labelLastDiv(bpmnId));
-    await expect(labelLastDivElementHandle.evaluate(node => node.innerHTML)).resolves.toBe(expectedText);
-  }
```

Comment:
⚠️  I suggest we restore the `expectLabel` as a method of the `PageTester` class and make it private (there is no need for it to be used outside the class).
The signature is now harder to understand as the `bpmnQuerySelectors` is passed to the function

⚠️ I also suggest that the logic for deciding whether or not we do the label check has to stay within the function.
The current implementation forces the caller to know if the check has to be done or not whereas we want the same behavior for all bpm elements.
This is inconsistent with other expects we do for both model, integration and e2e tests: everywhere else, we do the detection inside the function.

With the current implementation, we only see `expectedText && expectLabel` in `expectSequenceFlow`. The reason of doing this check is not obvious except if we now that all the diagrams we used in test have a label on shapes (whereas gateways and events have sometimes no labels) and not always on sequence flows.

Reply:
- I moved this function to be consistent with the other `expect` functions. 
- Moreover, I find it strange to use the function saying that we are waiting for a label, while we are not testing anything.

Reply:
so let's rename the `expectLabel` into something clearer like `checkLabel` to indicate that we don't always expect label existence.
Other expect functions are pure functions, that's why they are out of the class, this one depended on the some class properties, that's why it was defined in the class.